roles/openshift_common/meta/main.yml

```diff
@@ -13,4 +13,4 @@ galaxy_info:
   - cloud
 dependencies:
 - role: openshift_facts
-- role: openshift_repos
+- role: openshift_docker
```

Comment:
I'm not sure I'm a fan of making openshift_docker a dependency of openshift_common. This is going to force docker to be installed on hosts that may not need it. The existing conditional dependencies should handle the case of installing docker where necessary only.